gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -124,8 +124,8 @@ resources:
               export OS_PROJECT_ID="demo_project_id"
               export OS_PASSWORD="demo_password"
 
-              # set other region in NAME and URL if required
-              # available regions: cbk
```

Comment:
@dschwabeS11 For using the DBL region, auth URL should be `https://keystone.cloud.syseleven.net:5000/v3`

It will probably still work, but it is far from nice. Let's change OS_AUTH_URL everywhere to `https://keystone.cloud.syseleven.net:5000/v3`

Reply:
@stefreak Created another [PR](https://github.com/syseleven/heattemplates-examples/pull/50) for this.